packages/core/src/utils.ts

```diff
@@ -663,11 +665,15 @@ export const prettyFunctionCall = (
     removedBrackets + ' '.repeat(numSpacesForClosingParenthesis)
 
   const addressTag = address !== '' ? `<${address}>` : ''
+  const valueString =
+    BigInt(value) > BigInt(0)
+      ? `{ value: ${formatUnits(BigInt(value), 'ether')} ether }`
```

Comment:
I just used `ether` here instead of the actual currency for the network because I think the preview should closely resemble Solidity logic.